add: energy drink

WolfLox opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
Респрайт кофе но без передоза и зависимостей завёрнутые в банки и раскиданные по вендоматам по просьбе меланхолика. Спрайты от fantik
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
![image](https://github.com/ss220-space/Paradise/assets/108519426/b19faa1b-90cf-4360-b417-ba04b40655b8)
![image](https://github.com/ss220-space/Paradise/assets/108519426/8709f799-6e40-495f-825d-3a109de6b6ff)

<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений
![image](https://github.com/ss220-space/Paradise/assets/108519426/307afdf1-5a15-4323-bee1-1890535be2c0)
![image](https://github.com/ss220-space/Paradise/assets/108519426/2faf112c-1a1a-4b90-a744-7702183c18f5)
![image](https://github.com/ss220-space/Paradise/assets/108519426/b79a54c3-bc51-4504-8a58-c7aa37cbc56e)
![image](https://github.com/ss220-space/Paradise/assets/108519426/d44ee5bc-2628-48fc-b831-5f6215d566e3)
![image](https://github.com/ss220-space/Paradise/assets/108519426/427b8d13-fc70-41b7-b3c0-b4e32d232992)
![image](https://github.com/ss220-space/Paradise/assets/108519426/40f90f30-11a6-4261-9fd8-60594ba6a6bd)

<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->

Bizzonium committed:
server: byond tracy update (#4395)

WolfLox committed:
energy drink

Респрайт кофе но без передоза и зависимостей

WolfLox committed:
tweak

Правки по просьбе меланхолика

Etrnlmelancholy commented:
speed merge

TrashDoxx commented:
> speed merge

Молчать, педовка.

LiquidPotroh commented on code/modules/food_and_drinks/drinks/drinks/cans.dm:
```diff
@@ -174,6 +174,34 @@
+	desc = "The heart will say: - My stop."
+	icon_state = "heart_attack"
+	item_state = "heart_attack"
+	list_reagents = list("energetik" = 30)
```
Разрешили переводить билд, пихаем везде транслит :SeemsGood:

WolfLox replied:
Чтобы избежать повторения решил использовать транслит, зато по настоящему уникальный айди!

Etrnlmelancholy commented:
> > speed merge
> 
> Молчать, педовка.

Отменяю твою сессию на отстрапонивание Момокой завтра в 12:30

TrashDoxx commented:
> Отменяю твою сессию на отстрапонивание Момокой завтра в 12:30

Я мечтаю, чтобы девочки писяли мне в рот. Традиционно сабмиссивные игрища уже меня не привлекают.

THEROADCONEMAN commented:
![image](https://github.com/ss220-space/Paradise/assets/139582597/233aad7b-2f98-4cc9-a969-fbe245e02225)

Rerik007 committed:
bugfix: fix ashwalker & ashwalker runtime (#4518)

fix ashwalker & ashwalker runtime

LiquidPotroh committed:
bugfix: Eye of god isn't deleting when improving (#4517)

fix

Gottfrei committed:
bugfix: Library Computer Icons Fix (#4512)

Library Computer Icons Fix

Gottfrei committed:
bugfix: Canister Overlays Fix (#4510)

Canister Overlays Fix

ROdenFL committed:
add: loadout icons (#4507)

* bounty: loadout icons

* better way to do

Rerik007 committed:
bugfix: broken loadout of cargo items (#4509)

Update loadout_uniform.dm

Rerik007 committed:
bugfix: Keycard Authentication Devices are broken (#4516)

* Update keycard_authentication.dm

* Update code/modules/security_levels/keycard_authentication.dm

Co-authored-by: BeebBeebBoob <beebbeebboob@mail.ru>

---------

Co-authored-by: BeebBeebBoob <beebbeebboob@mail.ru>

Vladisvell committed:
bugfix: Toggling thunderdome enrolling state no longer causes runtimes (#4514)

Rerik007 committed:
bugfix: 3 -> 4 for CALTROP_BYPASS_ROBOTIC_LEGS (#4505)

* Update flags.dm

* now we check foots, not legs

Samirakis committed:
bugfix: borer respawn from "Respawn as NPC" (#4494)

Update mob.dm

Gottfrei committed:
bugfix: Hardsuit Fixes (#4496)

Hardsuit Fixes

Daeberdir committed:
bugfix: APC usability again. (#4486)

* Update apc.dm

* MEGAPROVERKA slaying.

* textiki

* Ahem.

* Deleting dead code.

Gottfrei committed:
bugfix: Monkey Emotes Fix (#4511)

Monkey Emotes Fix

aeternaclose committed:
bugfix: ShelterCapsuleFix (#4121)

* capsuleshelterfix

* major changes

* zuzya blyat

* zuzya blya 2

* zyza blyat 3

* zyza blyat 4

* momo prosila

Gottfrei committed:
add: Gorillas! OOGA! OOGA! OOGA! (#3969)

Gorillas-OOGA-OOGA-OOGA

aeternaclose committed:
add: CereImprovement-pt9 (#4178)

github-actions[bot] committed:
Automatic NanoMap Update (#4501)

SAADf603 committed:
Bugfix: Spacehotel shard steal (#4506)

actions-user committed:
Automatic changelog generation

Rerik007 committed:
bugfix: now hideheadseat for mask really hide headseats (#4519)

Ksaikok46 committed:
bugfix: ichorium now correctly costs 10 TC  (#4531)

Update gems.dm

Rerik007 committed:
bugfix: fix some give runtimes (#4528)

fix some give runtimes unregister

Rerik007 committed:
bugfix: fix runtime in spacebattle (#4522)

Update spacebattle.dm

Gottfrei committed:
bugfix: Respawnable Gorillas (#4534)

Respawnable Gorillas

Rerik007 committed:
bugfix: fix signal overriden runtime in xeno consoles (#4527)

Update xenobio_camera.dm

WolfLox committed:
part 2

Изменения согласно ТЗ

WolfLox committed:
tweak to second part

Цифры приведены к значениям ТЗ. Убран кусок кода использовавшийся потому что могу

Gottfrei committed:
bugfix: Correct Syndicate Animals Spawn (#4539)

Correct Syndicate Animals Spawn

BeebBeebBoob committed:
qol: Sets W.E.T. on janitor's holosigns to on default (#4477)

* qol: Sets W.E.T. on janitor's holosignd to on default

* Update holosign.dm

Gottfrei committed:
bugfix: Cryocell Overlay Fix (#4545)

Cryocell Overlay Fix

Gottfrei committed:
bugfix: Sawn Off Guns Icons Fix (#4544)

Sawn Off Guns Icons Fix

Rerik007 committed:
bugfix: fix nondamage projectile blood animation (#4549)

Update projectile.dm

ROdenFL committed:
bugfix: Cargo Console credits cost (#4550)

bugfix: ( and ) update

aeternaclose committed:
add: CereImprovement-pt10 (#4555)

Rerik007 committed:
bugfix: fixes access in quest console (#4554)

Gottfrei committed:
bugfix: Revolver Ammo Count Fix (#4553)

xTrogwaRx committed:
bugfix: normalize guitar volume (#4552)

Gottfrei committed:
bugfix: Wisp Lantern Icons Fix (#4548)

Wisp Lantern Icons Fix

Vladisvell committed:
bugfix: Clockwork armor is no longer rainbow error after enchanting (#4546)

Rerik007 committed:
bugfix: fixes respawn as npc verb (#4543)

* bugfix: fixes respawn as npc verb

* make npc list better

* i did how beep said

Vladisvell committed:
bugfix: Antag Paradise hotfix 4 (#4533)

* bugfix: Antag Paradise hotfix 4

* workaround: only humanoids can become antags now in processing antag paradise

ROdenFL committed:
bugfix: chat_box fixes (#4526)

* bugfix: chat_box fixes

* bugfix: chat box fixes2

Rerik007 committed:
bugfix: fix drop_on_fail logic in inventory (#4524)

Update inventory.dm

VasilevAY committed:
add: Biohazard bodybags (#4465)

github-actions[bot] committed:
Automatic NanoMap Update (#4536)

Ksaikok46 committed:
tweak: adrenaline for changelings (#4488)

Vladisvell committed:
refactor: antag checking optimization (#4521)

Vladisvell committed:
refactor: Optimized organ handling (#4523)

WolfLox committed:
add: glowsticks box in artvend (#4532)

Vladisvell committed:
bugfix: Localhost-configured admin access is now working (#4542)

actions-user committed:
Automatic changelog generation

github-actions[bot] committed:
Automatic NanoMap Update (#4557)

aeternaclose committed:
bugfix: Cerestation Disposal Fix (#4566)

VasilevAY committed:
bugfix: Fix typo in hazard bodybags icons (#4562)

Rerik007 committed:
bugfix: fixes "Cannot execute null.getRankAndAssignment()." runtime (#4565)

ROdenFL committed:
bugfix: rcd kill beach defib recursion (#4547)

actions-user committed:
Automatic changelog generation

Etrnlmelancholy commented:
@WolfLox выводи пр из драфта и фикси мердж конфликт зая

WolfLox committed:
just like coffe

to do: merge conflict

WolfLox committed:
Merge branch 'energy-drink-two' of https://github.com/WolfLox/Lamp-Lover into energy-drink-two

WolfLox commented:
![image](https://github.com/ss220-space/Paradise/assets/108519426/08ba33c9-3d5d-4e57-ba99-9a0ac5fc6ed7)
...
